content/en/releases/download.md

```diff
@@ -70,6 +72,11 @@ those derivations are signed in the same way as the multi-architecture manifest
 
 <!-- TODO: describe how to use the signed images -->
 
+Full list of images that are signed can be found [here](/examples/admin/signed-images/auto-generated-list-of-all-signed-images.txt)
```

Comment:
Preview: https://deploy-preview-32184--kubernetes-io-vnext-staging.netlify.app/releases/download/

nit: A period is missing.

```suggestion
Full list of images that are signed can be found [here](/examples/admin/signed-images/auto-generated-list-of-all-signed-images.txt).
```